app/views/applications/edit.html.erb

```diff
@@ -13,4 +13,9 @@
 
 <%= render partial: 'shared/subheader' %>
 
+<hr/>
+<h1>Notice:</h1>
+<h2>The Application's clientID should be prefixed by "beacon-manager-"</h2>
```

Comment:
Minor heads-up, nothing's wrong now so no need to change. When you define a separator for a prefix, avoid using it within the same prefix. This could be a source of issues down the road. For example, I see for now you're splitting by ```-```, and then use ```.last``` (safe for now, but I guess you see what I mean).

Reply:
Got it. For now I'll leave it like this, though in the future we might think of a better way to handle naming the client IDs on Keycloak